REFERENCE.md

```diff
@@ -273,7 +273,7 @@ unittest_main()
 ```
 
 
-# Build Scripts
+## Build Scripts
```

Comment:
I'm not sure I agree with the linter on this.  "Build scripts" and "Mocks of Arduino Hardware Functions" are major sections in this document, which is why the titles were formatted as such.

Reply:
The linter suggests to only have one top level header. I removed the changes and updated the configuration.